### ISSUE TITLE
[docs]: update repo status tables for all pattern pages

### ISSUE THREAD

**Issue by eyevana** [RedHat-UX/red-hat-design-system#1076]
We need to update the `repoStatus.csv` file for all Pattern pages that should be rendering a repo status table.

https://github.com/RedHat-UX/red-hat-design-system/pull/1057#discussion_r1235409243

**Comment by bennypowers** [comment]
Is this still relevant?

**Comment by coreyvickery** [comment]
> Is this still relevant?

@bennypowers Yes, but perhaps it would be easier for a developer to track this information down instead of me.

Or a larger conversation is needed about how exactly we keep track of all of our elements in patterns in different repos other than `RHDS`. I believe other design system websites feature this information on individual pages, I don't know how they do it, but it would be nice to fit in some automation.
